### DISCUSSION TITLE
Cluster-scoped gateway

### DISCUSSION THREAD

**Discussion by cabrinoob** [kubernetes-sigs/gateway-api#1988] [Q&A] [answered]
Hello, 
I have a little question regarding the `Gateway`. According to this schema :

![image](https://user-images.githubusercontent.com/5182775/235920125-d39c2676-f1a3-44fe-a79b-6f91ecb5c14b.png)

We can see a clear separation of responsibilities between Gateway and HTTPRoutes. That's a good thing, but, is it possible for the Infrastructure team to provide a Gateway in one namespace, and that the dev teams can deploy HTTPRoutes that use this Gateway in another namespaces ?

What I understand here is that if the TeamA deploys an HTTPRoute in NamespaceA, and the TeamB deploys another HTTPRoute in NamespaceB, and if we want those 2 HTTPRoute are using the same Gateway, I need to deploy the same Gateway in NamespaceA and NamespaceB.

Is that right? Or is it possible to point the `spec.parentRefs` of the HTTPRoute to a "common" namespace ?

Thank you for your explanations.

**Comment by youngnick** [comment, marked as answer]
The `spec.parentRefs` of the HTTPRoute (well, any Route) can be safely pointed to a Gateway in another namespace as long as the receiving Gateway also configures the `spec.allowedRoutes` section. (The default Gateway behavior is only to accept Routes from the same namespace, but that can be overridden using `spec.allowedRoutes`).

**Reply by cabrinoob** [reply]
Thank you for your reply
